patroni/dcs/__init__.py

```diff
@@ -778,16 +778,71 @@ def from_node(version: _Version, value: str) -> 'TimelineHistory':
         return TimelineHistory(version, value, lines)
 
 
+class Status(NamedTuple):
+    """Immutable object (namedtuple) which represents `/status` key.
+
+    Consists of the following fields:
+
+    :ivar last_lsn: :class:`int` object containing position of last known leader LSN.
+    :ivar slots: state of permanent logical replication slots on the primary in the format: ``{"slot_name": int}``.
```

Comment:
```suggestion
    :ivar slots: state of permanent replication slots on the primary in the format: ``{"slot_name": int}``.
```

My understanding is that once https://github.com/zalando/patroni/pull/2852 is merged, this will also store physical slots.

Maybe we could explain in the docstring the expected structure for logical and for physical slots?